docs/source/DeveloperGuide/TornadoServices/index.rst

```diff
@@ -0,0 +1,330 @@
+HTTPS Services with Tornado
+===========================
+
+.. contents::
+
+
+************
+Presentation
+************
+This page summarizes the changes between DISET and HTTPS. You can all also see these presentations:
+
+- `Presentation of HTTPS in DIRAC  <https://docs.google.com/presentation/d/1t0hVpceXgV8W8R0ef5raMK3sUgXWnKdCmJUrG_5LsT4/edit?usp=sharing>`_.
+- `Presentation of HTTPS migration <https://docs.google.com/presentation/d/1NZ8iKRv3c0OL1_RTXL21hP6YsAUXcKSCqDL2uhkf8Oc/edit?usp=sharing>`_.
+
+
+*******
+Service
+*******
+
+.. graphviz::
+
+   digraph {
+   TornadoServer -> YourServiceHandler [label=use];
+   YourServiceHandler ->  TornadoService[label=inherit];
+
+
+   TornadoServer  [shape=polygon,sides=4, label = "DIRAC.Core.Tornado.Server.TornadoServer"];
+   TornadoService  [shape=polygon,sides=4, label = "DIRAC.Core.Tornado.Server.TornadoService"];
+   YourServiceHandler  [shape=polygon,sides=4];
+
+   }
+
+Service returns to Client S_OK/S_ERROR encoded in JSON
+
+*********************************************************
+Important changes between DISET server and Tornado Server
+*********************************************************
+
+Internal structure
+******************
+
+- :py:class:`~DIRAC.Core.DISET.ServiceReactor` is now :py:class:`~DIRAC.Core.Tornado.Server.TornadoServer`
+- :py:class:`~DIRAC.Core.DISET.private.Service` and :py:class:`~DIRAC.Core.DISET.RequestHandler` are now merge into :py:class:`~DIRAC.Core.Tornado.Server.TornadoService`
+- CallStack from S_ERROR are deleted when they are returned to client.
+- Common config for all services, there is no more specific config/service. But you can still give extra config files in the command line when you start a HTTPS server.
+- Server returns HTTP status codes like ``200 OK`` or ``401 Forbidden``. Not used by client for now but open possibility for usage with external services (like a REST API)
+
+How to write service
+********************
+
+Nothing better than an example::
+
+  from DIRAC.Core.Tornado.Server.TornadoService import TornadoService
+  class yourServiceHandler(TornadoService):
+
+    @classmethod
+    def initializeHandler(cls, infosDict):
+      ## Called 1 time, at first request
+
+    def initializeRequest(self):
+      ## Called at each request
+
+    auth_someMethod = ['all']
+    def export_someMethod(self):
+      ## Insert your method here, don't forget the return should be serializable
+      ## Returned value may be an S_OK/S_ERROR
+      ## You don't need to serialize in JSON, Tornado will do it
+
+Writing a service for tornado and diset is similar. You have to define your method starting with ``export_``, and your initialization method is a class method called ``initializeHandler``.
+Main changes in tornado are:
+
+- Service are initialized at first request
+- You **should not** write a method called ``initialize`` because Tornado already use that name, so the ``initialize`` from diset handlers became ``initializeRequest``
```

Comment:
```suggestion
- You **should not** write a method called ``initialize`` because Tornado already use that name, so the ``initialize`` from DISET handlers became ``initializeRequest``
```